DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: nfidd
```

Comment:
do we need to change the package name so it doesn't conflict with the other nfidd package? Maybe it doesn't matter, but thought I'd ask.

Reply:
we shouldn't have to as there isn't a reason people should have both installed but we could do if want to